nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py

```diff
@@ -46,6 +48,9 @@ def go_to_patient_phase(self):
         # update to driving screen
         task_reponse = self.send_interface_request_task.execute(RobotStatus.DRIVING)
 
+        rospy.loginfo("Lowering body to home")
+        status = self.move_body_task.home()
```

Comment:
notice that the return is being stored and immediately overwrriten. so this line is the same as just `self.move_body_task.home()`.

not a big deal, but just in case you thought u were using the return code

Reply:
@le2tang I'm pushing this through but take note of this